elephant/cubic.py

```diff
@@ -0,0 +1,212 @@
+'''
+CuBIC is a statistical method for the detection of higher order of
+correlation in of parallel spike train based on the analysis of the
```

Comment:
"higher order of correlation" --> "higher order correlations"
"in of" --> "in"
"parallel spike train" --> "parallel spike trains"